src/modules/paymentProcessor/PP_Simple_v1.sol

```diff
@@ -258,4 +262,14 @@ contract PP_Simple_v1 is Module_v1, IPaymentProcessor_v1 {
         );
         return (success && data.length != 0 && _token.code.length != 0);
     }
+
+    function _validOriginAndDestinationChain(
```

Comment:
Why the mismatch between calling it "destination" here and "target" 2 lines below?